Process.c

```diff
@@ -707,6 +707,21 @@ void Process_printLeftAlignedField(RichString* str, int attr, const char* conten
    RichString_appendChr(str, attr, ' ', width + 1 - columns);
 }
 
+void Process_printPercentage(float val, char* buffer, int n, int* attr) {
+   if (val >= 0) {
+      if (val < 100) {
```

Comment:
Shadow out values below `0.1` to have interesting values (non-zero) values stick out more.

Reply:
Unrelated to the previous comment, but affects the same line:

The previous code used `99.9F` as limit; consider a value of `99.999`:
- currently it is bigger than `99.9F` and printed with `%3d. "` as `100. `.
- here it is smaller than `100` and printed with `"%4.1f "` it will print `100.0 `, which is one character too much for the columns